server/src/test/java/org/apache/iotdb/db/engine/modification/DeletionFileNodeTest.java

```diff
@@ -180,8 +180,12 @@ public void testDeleteInBufferWriteFile()
       assertTrue(directory.isDirectory());
       if (directory.isDirectory()) {
         for (File file : directory.listFiles()) {
-          if (file.getPath().endsWith(ModificationFile.FILE_SUFFIX)) {
-            modFiles.add(file);
+          if(file.isDirectory()){
```

Comment:
Please perform a reformat here and there.

Reply:
fixed~